minkindr/include/kindr/minimal/implementation/quat-transformation-inl.h

```diff
@@ -292,6 +294,19 @@ QuatTransformationTemplate<Scalar>::cast() const {
       getPosition().template cast<ScalarAfterCast>());
 }
 
+template<typename Scalar>
+QuatTransformationTemplate<Scalar> interpolateLinearly(
```

Comment:
This name is misleading and requires at least documentation. Here are the problems shat should be somehow addressed:
* It is not linear (the rotation part). That is why people use "spherical linear" (wired but at least not straight wrong ;) ).
* It could be confused with the exponential interpolation in SE3. What you do is exponential interpolation in SO(3)X R^3.
I would probably call it `interpolateComponentwiseExponentially` but most user would probably not understand. Maybe just `interpolateComponentwise` ? And then a little doc, that basically presents the impl in brief. ?
 
 
Renaming is probably